artifacthub/artifacthub-repo.yml

```diff
@@ -0,0 +1,6 @@
+# Configuration for our Artifact Hub repository
+# See https://github.com/artifacthub/hub/blob/master/docs/metadata/artifacthub-repo.yml
+repositoryID: e7f9f3ad-5080-4095-8ff9-98a0b71371b4
+owners:
+  - name: Tom Kerkhove
```

Comment:
Shouldn't we list here `KEDA Community`? Or something similar?

Reply:
What do you mean? In terms of owners? It has to be a contact person.

The repo ID is the ID on artifact hub itself.

Reply:
Does it need to be a person or an email? If email, shouldn't we use maintainers email here? If a person, than proceed with what you have here of course :)